src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -252,12 +252,25 @@ void testErrorReportedForMissingPresenterFieldAndOtherBlockWithValidationErrors(
         overseasEntitySubmissionDto.setManagingOfficersIndividual(null);
         overseasEntitySubmissionDto.setManagingOfficersCorporate(null);
 
-        Errors errors = overseasEntitySubmissionDtoValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT);
+        Errors errors = overseasEntitySubmissionDtoValidator.validateFull(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT);
         String qualifiedFieldName = PRESENTER_FIELD;
         String validationMessage = String.format(ValidationMessages.NOT_NULL_ERROR_MESSAGE, qualifiedFieldName);
         assertError(qualifiedFieldName, validationMessage, errors);
     }
 
+    @Test
+    void testErrorNotReportedForMissingPresenterFieldAndOtherBlockForPartialValidation() {
```

Comment:
I think we should have at least a couple more tests around the new 'partial' validation implementation as there are a few conditionals ('if' blocks) in that new code. Sonar should highlight which paths are and aren't tested.

Reply:
Added a couple more, hopefully it'll shut sonar up.

Reply:
Sonar was reporting that each conditional line had been hit but not all paths through the check logic had (e.g. testing with a null value and a non-null value). I think you've largely addressed this now though, so thanks.